pkl-core/src/test/files/LanguageSnippetTests/input/api/mathModule.pkl

```diff
@@ -130,6 +130,10 @@ examples {
     math.atan(math.tan(-2.34))
   }
 
+  ["atan2"] {
+    math.atan2(4, -3)
```

Comment:
Let's add some more examples that exercise all permutations.

```suggestion
    math.atan2(4, -3)
    math.atan2(4.5, -3.5)
    math.atan2(4.5, -3)
    math.atan2(4.5, 3.5)
```


Also: maybe a test for correspondence with PI, in the `facts` section:

```pkl
facts {
  ["atan2"] {
    math.atan2(1, 0) == math.pi / 2
    math.atan2(0, -1) == math.pi
  }
}
```